mypyc/test-data/run-floats.test

```diff
@@ -10,3 +10,11 @@ assert str_to_float("1.234567") == 1.234567
 assert str_to_float("44324") == 44324.0
 assert str_to_float("23.4") == 23.4
 assert str_to_float("-43.44e-4") == -43.44e-4
+
+[case testFloatAbs]
+def test_abs() -> None:
+    assert abs(0.0) == 0.0
+    assert abs(-1.234567) == 1.234567
+    assert abs(44324) == 44324.0
```

Comment:
This is also using an integer argument, so we don't actually any call with a positive value.